relay-aws-extension/README.md

```diff
@@ -26,6 +26,7 @@ export CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER=x86_64-unknown-linux-gnu-gcc
 rustup target add x86_64-unknown-linux-gnu
 
 # Install objcopy (used in `Makefile`)
+# Make sure to change `2.37` to the specific version you installed (use: brew info binutils)
```

Comment:
Instead of doing this, would it make more sense to update the command below to:

```shell
ln -s "$(brew --prefix binutils)/bin/gobjcopy" /usr/local/bin/objcopy
```